.changeset/nice-tips-hide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lg-tools/storybook-decorators': patch
+---
+
+Removes `overflow: auto` from `rootStyles`.
```

Comment:
This prevents automatic scrolling?

Reply:
This was preventing position sticky from working. I don't think this breaks anything else.